leveler/leveler.py

```diff
@@ -74,6 +74,7 @@ def __init__(self, bot: Red):
         }
         default_guild = {
             "disabled": False,
+            "antifarmrep": False,
```

Comment:
That's not really selfdescriptive setting name.

```suggestion
            "rep_rotation": False,
```